lib/reverse_sentence.rb

```diff
@@ -1,6 +1,47 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+
+# Time complexity: ? O(3n) == O(n) , using 3 while loops
+# Space complexity: ? O(n)
+
 def reverse_sentence(my_sentence)
```

Comment:
⚠️ This works, although it requires the external array you create and so doesn't do the work in place.  Think about how you could do this in-place without needing the external array `new_arr`.